src/main/java/jalau/at18/kingoftokyo/view/rolldicesection/KeepDiceSectionUI.java

```diff
@@ -36,18 +39,25 @@ public void initializeDice() {
         add(displayResult);
     }
 
-    public RollerButton getRollerButton() {
-        return rollerButton;
+    public void resetUI() {
+        keepButton.setVisible(false);
+        for (DiceFaceLabel diceFaceLabel : listKeepDiceFaceLabel) {
+            diceFaceLabel.painQuestionMark();
+        }
     }
 
-    public JLabel getDisplayResult() {
-        return displayResult;
+    public JButton getKeepButton() {
+        return keepButton;
     }
 
     public List<DiceFaceLabel> getListKeepDiceFaceLabel() {
         return listKeepDiceFaceLabel;
     }
 
+    public JLabel getDisplayResult() {
+        return displayResult;
+    }
+
     public void sendResults(int[] effect) {
```

Comment:
this is the method that I thought that it is missing, but it is here....then all ok.

Reply:
Yes, when I did the rebase I saw this update and I kelt this.